titan-compiler/coder.lua

```diff
@@ -912,7 +1001,17 @@ generate_exp = function(exp) -- TODO
 
         local op = exp.op
         if op == "#" then
-            error("not implemented yet")
+            local tmp = tmp_name()
```

Comment:
What happens if we use the length operator with a string?

Reply:
Something horrible would happen. Good catch!